test/types/basic_schema.test-d.ts

```diff
@@ -20,7 +20,7 @@ expectType<Collection<ACounterWithId>>(new Collection<ACounterWithId>(db, ''));
 ////////////////////////////////////////////////////////////////////////////////////////////////////
 // Simple Schema that does not define an _id
 // With _id
-type InsertOneArgOf<S> = Parameters<Collection<S>['insertOne']>[0];
+type InsertOneArgOf<S extends Document> = Parameters<Collection<S>['insertOne']>[0];
```

Comment:
My local version of tsd was updated which pulls in a new TS version to test against and this is a requirement since TS 4.8 figured I could just preemptively fix this.